save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CreationView.kt

```diff
@@ -104,7 +104,7 @@ enum class GitConnectionStatusEnum {
 @OptIn(ExperimentalJsExport::class)
 class CreationView : RComponent<PropsWithChildren, ProjectSaveViewState>() {
     private val fieldsMap: MutableMap<InputTypes, String> = mutableMapOf()
-    private lateinit var responseFromCreationProject: Response
+    private var responseFromCreationProject: Response? = null
```

Comment:
lateinit here caused a clash with a courutines, I do not have any ideas why by for now

Reply:
My idea - after execution of the main coroutine, `invokeOnCompletion` somehow still had a stale copy of `responseFromCreationProject` variable, which was unitialized. I think `invokeOnCompletion` is not needed here at all, because we can get response and immediately perform redirect from within coroutine body.

Reply:
no, that does not help :)
there is a failure on the "post" part